spec/03-plugins/02-udp-log/01-udp-log_spec.lua

```diff
@@ -197,9 +197,10 @@ for _, strategy in helpers.each_strategy() do
         },
         opts = {
           ["-authority"] = "udp_logging_grpc.test",
+          ["-H"] = "'Content-Type: text/plain'",
```

Comment:
just curious why would this fix the test?

Reply:
I don't remember, I think it was because of the `grpcurl` error message that prompted me.